main.tf

```diff
@@ -1 +1,27 @@
-# Terraform code goes here
+resource "digitalocean_loadbalancer" "main" {
```

Comment:
Few options are missing ie: `algorithm` and `redirect_http_to_https` that sound useful. But we can add them when actually needed, not to bloat too much now.